### PR TITLE
iwlist: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/iwlist.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# iwlist
+
+> Get more detailed wireless information from a wireless wireless_interface.
```
Comment (gutjuri) [inline pages/linux/iwlist.md:3]:
```suggestion
> Get detailed information from a wireless interface.
```

File: pages/linux/iwlist.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- List all the WPA encryption keys set in the device:
+
+`iwlist {{wireless_interface}} wpakeys`
+
+- List the encryption key sizes supported and list all the encryption keys set in the device:
+
+`iwlist {{wireless_interface}} keys`
+
+- List the various Power Management attributes and modes of the device:
```
Comment (gutjuri) [inline pages/linux/iwlist.md:30]:
```suggestion
- List the various power management attributes and modes of the device:
```

File: pages/linux/iwlist.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`iwlist {{wireless_interface}} power`
+
+- List the Generic Information Elements set in the device (used for WPA support):
```
Comment (gutjuri) [inline pages/linux/iwlist.md:34]:
```suggestion
- List generic information elements set in the device (used for WPA support):
```

File: pages/linux/iwlist.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Give the list of available frequencies in the device:
+
+`iwlist {{wireless_interface}} frequency`
+
+- List the bit-rates supported by the device:
+
+`iwlist {{wireless_interface}} rate`
+
+- List the WPA authentication parametes curently set:
```
[the remark this answers is not in the document]
Comment (fazlearefin) [reply pages/linux/iwlist.md:18]: Even the official doc have these spelling errors!

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**